recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - pybind11
     - hdf5 =1.10.5
     - gsl
-    - boost
+    - boost =1.74.0
```

Comment:
Don't explicitly set the boost version but use the migrator PR that will be opened by the bot.

Reply:
@xhochy Thank you for your comment.
I withdraw this pull-request.